documentation/modules/ROOT/pages/index.adoc

```diff
@@ -81,24 +75,19 @@ xref:python-sdk-guide:ROOT:index.adoc[Python SDK]
 [.icon-cli]
 --
 [.title]
-JS SDK Aspect Model Loader
+JavaScript SDK
 
 [.text]
-The JS SDK Aspect Model Loader offers functionality which helps software developers to work with Aspect Models 
-in their Typescript applications.
+Build UIs better and faster. This SDK loads Aspect Models and generates UI code, such as tables that unite aspect data & model.
```

Comment:
"UI code, such as tables that unite aspect data & model."
single comma might be wrong? Two or none?

Reply:
You mean to embed "such as" in commas? No, that is only sth you'd do for "e.g." and "i.e." -- they go with commas before and after. 👍

Yes, if the "such as"-clause was restrictive, it would go completely without commas. 

But as the core meaning of the sentence would not be changed by leaving the such-as part out, it is non-restrictive. Hence it is separated from the main clause by a comma. See also https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Commas/faq0025.html.

Thanks for paying attention to commas! ❤